edge-modules/edgehub-proxy/src/main.rs

```diff
@@ -200,23 +195,19 @@ fn run() -> Result<(), Error> {
     Ok(())
 }
 
-fn client(url: &Url) -> Result<APIClient, Error> {
-    let hyper_client = Client::builder().build(UrlConnector::new(&url)?);
-    let base_path = get_base_path(url);
+/*
+fn client(url: &Url) -> Result<WorkloadClient, Error> {
```

Comment:
Is this commented out function still needed? if not delete it

Reply:
Not needed. Commented out everything from before since it was a draft PR and work in progress but forgot to take them out permanently before sharing this